tests/Unit/SubPageList/Lister/RedirectCreator.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Tests\Unit\SubPageList\Lister;
+
+use Title;
+
+class RedirectCreator {
+
+	public function createRedirect( Title $title, $dest ) {
```

Comment:
I think abbreviating such as this does not pay off, and prefer `$destination`. As a type hint cannot be added for this parameter before PHP 7, adding a dock block that specifies it is a string is nice.